supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesRuntimeAttributes.scala

```diff
@@ -82,3 +90,28 @@ class DockerWorkingDirValidation extends StringRuntimeAttributesValidation(TesRu
   }
 }
 
+/**
+  * Validates the "preemptible" runtime attribute as a Boolean or a String 'true' or 'false', returning the value as a
+  * `Boolean`.
+  *
+  * `instance` returns an validation that errors when no attribute is specified.
+  *
+  * `configDefaultWdlValue` returns the value of the attribute as specified by the
+  * reference.conf file, coerced into a WomValue.
+  *
+  * `default` a validation with the default value specified by the reference.conf file.
+  */
+
+object PreemptibleValidation {
```

Comment:
we have a preemptible validation code for Google already -- if that helps to reference:
https://github.com/broadinstitute/cromwell/blob/0f4cf8779177171e12647d4b0de8c94438c3158d/supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiRuntimeAttributes.scala#L59

Reply:
The [TES specification](https://github.com/ga4gh/task-execution-schemas) approved by GA4GH uses a Boolean. The philosophy was to go with the common denominator set of features that a backend might support. I agree that a notion of preemtible retries is useful, but this is not part of the approved TES spec and making such a change proposal will require a round of approval in the GA4GH. I would suggest that the change to the Cromwell TES backend proceed as is for now.

Reply:
I didn't realize it needed approval, and I'm happy to bring this up with the GA4GH pull request directly.